ui/components/multichain/app-header/app-header.scss

```diff
@@ -0,0 +1,52 @@
+.multichain-app-header {
+  flex-flow: column nowrap;
+  z-index: 55;
```

Comment:
We should probably be storing our z-index's in a CSS variable and standardizing them? Where does this value come from or is it from the old header?

Reply:
That is a really nice suggestion. I used `55` here since we had already used it in `Dropdown` in file `ui/components/app/dropdowns/dropdown.js` and it's used directly in the header.